### ISSUE TITLE
OSM import not working for some symbols

### ISSUE THREAD

**Issue by dfgeorge** [OpenOrienteering/mapper#998]
### Steps to reproduce
1. Attached file has osm data for cobbler creek conservation park area, south australia: https://www.openstreetmap.org/export#map=14/-34.7873/138.7178
2. crt file attached has rule 403 leisure = park
3. map file attached has 4 areas marked 1 .. 4, all have leisure = park when checked with QGIS
  Area 1 converts OK. If I change the rule to 401 leisure = park the symbol changes
  Area 2 does not convert. If I select the area and convert manually it has been drawn with the wrong shape
  Area 3 does not convert. If I select the area and convert manually it has the correct shape
  Area 4 does is defined a 2 parts in osm (does both sides of the main road), this does not import. I have seen similar behaviour for areas with holes in too.

### Configuration
Mapper Version: 0.8.20171004
Operating System: Windows 7 32bit

**Comment by dfgeorge** [comment]
Files: [cobbler.zip](https://github.com/OpenOrienteering/mapper/files/1437475/cobbler.zip)

**Comment by dg0yt** [comment]
The OSM file has 443 occurences of leisure=park, the import only 1...

**Comment by dfgeorge** [comment]
Attached is a smaller example, from the reserve at 3 on the last map. Contains 1 leisure = park, nothing on map?
[test2.zip](https://github.com/OpenOrienteering/mapper/files/1440328/test2.zip)

**Comment by dg0yt** [comment]
I'm still working on the first example.
- Area 1 is okay anyway.
- At 2, there is an oval area with "leisure=pitch". In the OSM data, there is a surrounding polygon with "leisure=park" which does import properly with the Mapper versions I tested.
- At 3 and 4, I don't see an area in your map, but I get one when I import from OSM. However, these objects are covered by the "Salisbury East" polygon. Maybe we need to suppress administative boundaries?

**Comment by dg0yt** [comment]
There are too few tags on some objects. Your test2 map has

~~~
name           447143684
~~~
which is the id of an OSM way with no tags,
where as my import of your OSM data has

~~~
leisure        park
name           Martindale Reserve
osm_id         6641719
osm_timestamp  2016/10/12 06:13:20+00
other_tags     "type"=>"multipolygon"
~~~

which is the ID of an OSM multipolygon consisting of OSM way 447143684. So you are missing the multipolygons.

Please verify that the settings point to a valid and complete osmconf.ini. Mapper comes with [mapper-osmconf.ini](https://raw.githubusercontent.com/OpenOrienteering/mapper/master/src/gdal/mapper-osmconf.ini) which selects all relevant OSM keys for all types of objects. (To reset the relevant settings field, clear the table cell displaying "OSM_CONFIG_FILE").

**Comment by dfgeorge** [comment]
I checked OSM_CONFIG_FILE, it was pointing to the correct file but appears it was not using it. I clicked on it (as if to edit) then clicked OK. After that the import worked! 

Before I did this the import was not using  the CRT, OSM data was imported normally (all grey lines) then I was applying the CRT by using symbols -> Load CRT file. Now the import does the conversion.

I have the latest unstable installed on a Windows 10 box and a Linux box. They both were doing the same, same fix fixed both.

Thanks,
